core/block_svg.ts

```diff
@@ -671,6 +671,26 @@ export class BlockSvg
     }
   }
 
+  /**
+   * Add a class to the SVG group of this block.
+   *
+   * @param className
+   * @internal
```

Comment:
```suggestion
```

We actually want these to be public! You can remove the `@internal` tags from `addClass` and `removeClass`.

Reply:
Absolutely! Updated.